### DISCUSSION TITLE
In doc, why is the same field with two unique?

### DISCUSSION THREAD

**Discussion by gutbk201** [nextauthjs/next-auth#6067] [Help]
https://next-auth.js.org/adapters/prisma#setup
```
model VerificationToken {
  identifier String
  token      String   @unique
  expires    DateTime

  @@unique([identifier, token])
}
```
Why putting `@@unique([identifier, token])`? Since token is already unique?
Assume it's for multiple device, then should it be `token String`, allowing same token but different identifier?

**Comment by deibyrios** [comment]
This is a good question. If token is already unique, that gurantees that I can't have the same [identifier, token] in two different rows, so why is the @@unique([identifier, token]) used here?
